docs/networking/deep-dive.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 3
+sidebar_label: Deep Dive
+title: ""
+keywords:
+- Harvester
+- Networking
+- Topology
+---
+# Harvester Network Deep Dive
+
+The network topology below reveals how we implement the Harvester network.
+
+![](/img/v1.1/networking/topology.png)
+
+The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `oob`.
```

Comment:
```suggestion
The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `OOB`.
```

Reply:
In the diagram, the bridge name is `oob-br` and the bond name is `oob-bo`, so the cluster network is exactly `oob`, not `OOB`.